src/Microsoft.Dnx.Tooling/Publish/PublishProject.cs

```diff
@@ -564,9 +569,7 @@ private bool GenerateWebConfigFileForWwwRootOut(PublishRoot root, Runtime.Projec
             var attributesToOverwrite = new[]
             {
                 "processPath",
-                "arguments",
-                "stdoutLogEnabled",
-                "stdoutLogFile"
```

Comment:
Because we can leave a web.config in that folder so that dnu copies stuff over. Currently it copies some IIS config stuff like error page settings.